src/material-experimental/mdc-theming/prebuilt/indigo-pink.scss

```diff
@@ -1,6 +1,9 @@
 @import '../all-theme';
 @import '../../mdc-typography/all-typography';
 
+// Include non-theme styles for core.
+@include mat-core();
+
```

Comment:
Is this actually intentional? I was under the impression that these themes should be used in combination with the standard themes from Angular Material. Right now, if one uses experimental components while using other parts of Angular Material and uses both prebuilt themes, core styles are duplicated.

Reply:
@andrewseguin You might have a better response than I do.

If we don't include this it becomes mandatory to use both prebuilt themes in order to use mdc components. I think it is intuitive that including both prebuilt themes would have duplicate code.

As a counter point, people who already include both of these won't be thrilled if this duplicate code increases their package size by too much, so I don't know what the correct answer is here

Reply:
Since this is a prebuilt theme, I would expect it to be standalone and include `mat-core` styles. If a user only wants to use MDC components, they would see their app break with the current state of the prebuilt theme. This seems like the right change to me. Users may get unnecessary styles from the duplicated `mat-core`, but that means they would be importing two prebuilt themes. If they care about size, I would hope they know they should be individually pulling in the theme mixins for only the components they are using.

Reply:
Yeah, I'd agree with the sentiment that people who care about size should use individual fine-grained Sass mixins. 

Resolving this. I think either way is fine, and this one might be more convenient for standalone MDC component consumers.